docs/i18n/ja-jp/docusaurus-plugin-content-docs/current/getting-started.mdx

```diff
@@ -9,11 +9,7 @@ lang: ja-jp
 
```

Comment:
@seratch I tried combining the Socket Mode and HTTP versions. Can you read it to make sure I did so properly?